lib/jekyll/jekyll-feed.rb

```diff
@@ -47,5 +50,30 @@ def feed_exists?
         File.exist? Jekyll.sanitized_path(@site.source, path)
       end
     end
+
+    # Path to the feed.xslt.xml template file
+    def xslt_source_path
+      File.expand_path "../feed.xslt.xml", File.dirname(__FILE__)
+    end
+
+    def xslt_content
+      xslt_path = "feed.xslt.xml"
+      xslt = PageWithoutAFile.new(@site, File.dirname(__FILE__), "", xslt_path)
+      xslt.content = File.read(xslt_source_path).gsub(/(?<!\")\s+([<{])/, '\1')
+      xslt.data["layout"] = nil
+      xslt.data["sitemap"] = false
+      xslt.output
+      xslt
+    end
+
+    # Checks if a feed stylesheet exists in the site source
+    def xslt_exists?
```

Comment:
Should this an the `feed_exists?` methods be abstracted into a `file_exists?(file)` method could be shared for both?